README.md

```diff
@@ -192,6 +192,8 @@ OPTIONS
   -f, --file=file              (required) file location of API to update
   -h, --help                   show CLI help
   --visibility=public|private  [default: private] visibility of API in SwaggerHub
+  --publish                    publish the API version when it gets updated
```

Comment:
I think this needs to be updated again to align with the description in the class